generators/createDisplayUnitSet/templates/shared/index.hbs

```diff
@@ -19,7 +20,7 @@
     <div class="content fullscreen">
       <div class="title_container">
         <div class="title">
-          {{content.text}}
+          {{content.title}}
```

Comment:
<div class="copy">
          {{content.copy}}

Reply:
Same thing here - title or copy is ok with me